Makefile

```diff
@@ -99,19 +99,13 @@ sdist_name:=toil-$(dist_version).tar.gz
 docker_tag:=$(shell $(python) version_template.py dockerTag)
 default_docker_registry:=$(shell $(python) version_template.py dockerRegistry)
 docker_path:=$(strip $(shell which docker))
-ifdef docker_path
-    ifdef docker_registry
-        export TOIL_DOCKER_REGISTRY?=$(docker_registry)
-    else
-        export TOIL_DOCKER_REGISTRY?=$(default_docker_registry)
-    endif
+
+ifdef docker_registry
+    export TOIL_DOCKER_REGISTRY?=$(docker_registry)
 else
-    $(warning Cannot find 'docker' executable. Docker-related targets will be skipped.)
-    export TOIL_DOCKER_REGISTRY:=
+    export TOIL_DOCKER_REGISTRY?=$(default_docker_registry)
 endif
 export TOIL_DOCKER_NAME?=$(shell $(python) version_template.py dockerName)
-# Note that setting TOIL_DOCKER_REGISTRY to an empty string yields an invalid TOIL_APPLIANCE_SELF
```

Comment:
This seems like a useful comment. Why remove it?

Reply:
I tore this out because, as above, it serves no purpose anymore and is setting `TOIL_APPLIANCE_SELF` to an invalid string on purpose as a signal to skip docker tests (redundantly, so it doesn't actually need to do this to skip docker tests).  

Since this checks that `TOIL_APPLIANCE_SELF` is valid, this is painfully unhelpful.